### PR TITLE
Added new function "lerp" for linear interpolation and added skeleton for "sign" 

### PR DESCRIPTION
**Pull request by LSBOSS**
Make sure these boxes are checked before submitting this pull request~!~ Thank you!!

- [x] The test of the method you have fixed is passing
- [x] You have written a new skeleton method for someone else to work on!

### CODE REVIEW

File: test/sign.test.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+test("Sign of a postive number !=0 should be 1", () => {
+    expect(sign(123)).toBe(1);
+});
+
+test("Sign of a NaN should be NaN", () => {
+    expect(sign(NaN)).toBe(NaN);
+});
+
+test("Sign of a -0 should be -0", () => {
+    expect(sign(-0)).toBe(-0);
```
Comment (andreasgruenh) [inline test/sign.test.js]: I don't think this test will work or does it? Since 0 === -0, the `toBe(-0)` call will yield true for 0 and -0

**Comment by andreasgruenh** [comment]
Additionally, some linting rules are failing. Please run `yarn run lint` and fix those errors.

File: test/sign.test.js (hunk 1 above)
Comment (andreasgruenh) [reply test/sign.test.js]: Maybe we could add another method `isNegativeZero` and `isPositiveZero` and use these in the tests and test them in their own tests :D

Comment (BlakeGuilloud) [reply test/sign.test.js]: image

Comment (LSBOSS) [reply test/sign.test.js]: @BlakeGuilloud : Hahaha nice :)

@andreasgruenh: I looked at Math.sign and this function has those 5 return values I wrote test for. So let me do the following: 

* I fix the linting problems
* Remove both -0 and +0 tests 
* Write two new skeleton functions isNegativeZero and isPositiveZero for someone else to work on

**Comment by andreasgruenh** [comment]
Hey man, thanks for making these changes! Great job!

**Pull request merged**